feedhenry/fh-mbaas-express
# update module to support node10

* Update some of the deps to the latest version
* Get rid of turbo-test-runner

To verify:

* Checkout this PR and run `npm install` with node 10. Everything should work and there should be no mention of `bunyan` or `dtrace_provider` in the output.
